corehq/apps/users/role_utils.py

```diff
@@ -66,3 +102,12 @@ def initialize_domain_with_default_roles(domain):
             permissions=permissions_fn(),
             is_commcare_user_default=role_name is UserRolePresets.MOBILE_WORKER,
         )
+
+
+def add_program_manager_role_to_domain(domain):
+    role_name = UserRolePresets.PROGRAM_MANAGER
```

Comment:
Program Manager sounds like a very generic role name that is valid outside of ATE context. If this role is supposed to Manage ATE features why don't we call it Attendance_manager or something like that? This becomes important when we try to GA.

Reply:
Added as a [comment to spec](https://docs.google.com/document/d/1hU2SQvnfkfxu3RoxhRCKL6IuWl6zO7nuZFEbgF5grrE/edit?disco=AAAAo8HstNg) instead since this is a design question.